fuzz/ocsp_corpus/01cd28b0c1554e1a7207586d00869269eb08c66c

```diff
@@ -0,0 +1 @@
+����������
```

Comment:
Does this file just include binaries like `fuzz/ocsp_corpus/01f48d134e3943f77accb2ec7595d4bd6791139a`, which is marked by GitHub with `Binary file not shown.`?